[gamemodes]/[play]/play/play_players.lua

```diff
@@ -0,0 +1,35 @@
+local spawnsCount = #playerSpawns
+local skinsCount = #playerSkins
+
+function playSpawnPlayer(playerElement)
+	if isElement(playerElement) then
+		local randomSpawn = math.random(1, spawnsCount)
```

Comment:
The 1 is unnecessary.
```suggestion
		local randomSpawn = math.random(spawnsCount)
```